crates/stellar-scaffold-test/src/common.rs

```diff
@@ -342,7 +342,7 @@ pub fn find_binary(name: &str) -> Option<PathBuf> {
 pub fn find_stellar_wasm_dir() -> Option<PathBuf> {
     let exe_path = env::current_exe().ok()?;
     let project_root = find_project_root(&exe_path)?;
-    Some(project_root.join("target").join("stellar"))
+    Some(project_root.join("target").join("stellar/standalone"))
```

Comment:
Also why standalone here and local above?

Reply:
I pushed some changes with a wider replace.